roundwared/recording_collection.py

```diff
@@ -281,3 +297,68 @@ def _update_playlist_timed(self):
         self.playlist_timed = self.order_assets(self.playlist_timed)
 
         # logger.debug("Found timed assets: %s" % self.playlist_timed)
+
+
+    def _assets_by_user(self, user_id):
```

Comment:
This can maybe be optimized into a single query. I'm not sure its worth spending time on now, but I would keep an eye on this function in production and see if it causes a bottleneck.